sql/database.sql

```diff
@@ -10900,3 +10521,533 @@ CREATE TABLE IF NOT EXISTS `medex_recalls` (
   PRIMARY KEY (`r_ID`),
   UNIQUE KEY `r_PRACTID` (`r_PRACTID`,`r_pid`)
 ) ENGINE=InnoDB;
+
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `form_eye_base`
+--
+
+CREATE TABLE IF NOT EXISTS `form_eye_base` (
+  `id`         bigint(20) NOT NULL COMMENT 'Links to forms.form_id',
+  `date` datetime DEFAULT NULL,
+  `pid`        bigint(20)   DEFAULT NULL,
+  `user`       varchar(255) DEFAULT NULL,
+  `groupname`  varchar(255) DEFAULT NULL,
+  `authorized` tinyint(4)   DEFAULT NULL,
+  `activity`   tinyint(4)   DEFAULT NULL
+) ENGINE = InnoDB;
+ALTER TABLE `form_eye_base`
+  ADD PRIMARY KEY `form_link` (`id`);
+ALTER TABLE `form_eye_base`
+  CHANGE `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT;
```

Comment:
These ALTER TABLE statements should be removed and these things should be included in the CREATE TABLE call. Also should do the same for all the other tables.

Reply:
Will need guidance here.

Reply:
Can't you just do what you did in the old form_eye_mag table regarding the id column and the primary key?